publish-imgs

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 # vim: ft=sh sw=2 et
 # shellcheck shell=bash
-set -efo pipefail
+set -fo pipefail
```

Comment:
removing the `-e` here seems to break when i test locally

test with no `-e`
```
#!/usr/bin/env bash

set -fo pipefail

TEST=$(fake)
for i in ${TEST[@]}; do
	echo "in the loop $i"
done

echo "break past"
```
results in 
```
$ bash /tmp/test1.sh 
/tmp/test1.sh: line 5: fake: command not found
break past
```
vs with `-e`
```
#!/usr/bin/env bash

set -efo pipefail

TEST=$(fake)
for i in ${TEST[@]}; do
	echo "in the loop $i"
done

echo "break past"
```
which results in expected
```
$ bash /tmp/test1.sh 
/tmp/test1.sh: line 5: fake: command not found
```

what was the intent of removing `-e`?